jetson/nav/stateMachine.cpp

```diff
@@ -489,7 +489,7 @@ string StateMachine::stringifyNavState() const
 // Returns true if an obstacle is detected, false otherwise.
 bool StateMachine::isObstacleDetected() const
 {
-    return mPhoebe->roverStatus().obstacle().detected;
+    return mPhoebe->roverStatus().obstacle().distance >= 0;
```

Comment:
I am in favor of abstracting this out into a function, as long as you have it that way in simpleAvoidance, just to keep it consistent.